doc/source/ipynb/ufo.rst

```diff
@@ -108,7 +103,7 @@ than 1 will crash the reconstruction.
 
 .. code:: python
 
-    recon = tomopy.recon(proj, theta, center=center, algorithm=ufo.tomopy.fbp, ncore=1)
+    recon = tomopy.recon(proj, theta, center=center, algorithm=ufo_fbp, ncore=1)
```

Comment:
@matze Could you please check if the UFO reconstruction example in docs is using the same notation?

Reply:
This *is* the example in the documentation or do I misunderstand something?

Reply:
Ah. My confusion. Sorry.